server/base.py

```diff
@@ -71,9 +71,10 @@ def _updateJob(event):
     Called when a job is saved, updated, or removed.  If this is a large image
     job and it is ended, clean up after it.
     """
-    global JobStatus
+    global JobStatus, Job
     if not JobStatus:
         from girder.plugins.jobs.constants import JobStatus
+        from girder.plugins.jobs.models.job import Job
```

Comment:
I don't think this conditional is doing anything useful.  I think you can just remove the `Job` and `JobStatus` globals.